projects/aws/eks-anywhere/Makefile

```diff
@@ -17,8 +17,12 @@ COLLECTOR_IMAGE_NAME=eks-anywhere-diagnostic-collector
 COLLECTOR_IMAGE=$(IMAGE_REPO)/$(COLLECTOR_IMAGE_NAME):$(IMAGE_TAG)
 COLLECTOR_LATEST_IMAGE=$(IMAGE_REPO)/$(COLLECTOR_IMAGE_NAME):latest
 
+.PHONY: fetch-binaries
+fetch-binaries:
+    build/fetch_binaries.sh $(RELEASE_BRANCH) $(RELEASE) $(EKS_DISTRO_RELEASE_VERSION)
```

Comment:
I think this needs to be a tab.

Reply:
We don't seem to be using these vars, can you remove them from here as well as in the Makefile under `projects/aws/eks-anywhere-build-tooling`?

Reply:
yep I was wondering about that

Reply:
ok removed from /eks-anywhere/Makefile